package.json

```diff
@@ -24,21 +24,21 @@
     "run-example-vite": "cd examples/example-vite-app && npm run preview",
     "test-on-examples": "./test-on-examples.sh",
     "unused-exports": "ts-prune --error --ignore 'src/index.ts'",
-    "lint": "xo && tsc --noEmit && eslint '*/**/*.ts' && npm run unused-exports",
-    "lint-fix": "tsc --noEmit && eslint '*/**/*.ts' --fix && xo --fix",
+    "lint": "xo && npm run unused-exports",
+    "lint-fix": "xo --fix",
     "lint-staged": "lint-staged",
-    "prepare": "husky install",
+    "prepare": "husky",
     "release": "np --no-publish",
     "version": "./update-versions.sh",
     "dev": "ts-node src/bin",
     "start": "NODE_ENV=production node dist/bin.js"
   },
   "engines": {
-    "node": ">=18"
```

Comment:
@chriskalmar Node 22 is quite recent. Is it really necessary to have that as a minimum version, in the `engines` section? 
Do you really use features from the newest node engine that aren't available in Node 18? 
Thanks.

Reply:
I hear you @n2o1988. Node 22 was maybe a bit too soon 😅 
There are some dependencies that are sitting on at least Node 20.
Let me have a look into it, maybe Node 18 is still manageable (for the few more months it still is in maintenance).

Reply:
fixed: https://github.com/lost-pixel/lost-pixel/pull/428